components/popover/popover-mixin.js

```diff
@@ -212,27 +607,130 @@ export const PopoverMixin = superclass => class extends superclass {
 
 	}
 
-	_handleFocusTrapEnter() {
-		this._focusContent(this._getContentContainer());
+	#handleFocusTrapEnter() {
+		this.#focusContent(this.#getContentContainer());
 
 		/** Dispatched when user focus enters the popover (trap-focus option only) */
 		this.dispatchEvent(new CustomEvent('d2l-popover-focus-enter', { detail: { applyFocus: this._applyFocus } }));
 	}
 
-	_removeAutoCloseHandlers() {
-		this.removeEventListener('blur', this._handleAutoCloseFocus, { capture: true });
-		document.body?.removeEventListener('focus', this._handleAutoCloseFocus, { capture: true }); // DE41322: document.body can be null in some scenarios
-		document.removeEventListener('click', this._handleAutoCloseClick, { capture: true });
+	#handleResize() {
+		this.resize();
 	}
 
-	_renderPopover() {
-		const content = html`<div class="content"><slot></slot></div>`;
+	async #position(contentRect, options) {
```

Comment:
This logic is the same as `DropdownContentMixin` with some renaming. Was also nice to ditch the non-fixed positioning logic.